pkg/infra/iac3/templates/aws/ecs_task_definition/container_definitions.ts.tmpl

```diff
@@ -0,0 +1,103 @@
+[
+    {{- range $i, $cd := . }}
+    {
+        {{- if $cd.Command }}
+        command: [
+        {{ range $i, $cmd := $cd.Command }}
+            "{{ $cmd }}",
+        {{- end }}
+        ],
+        {{- end }}
+        {{- if $cd.Cpu }}
+        cpu: {{ $cd.Cpu }},
+        {{- end }}
+        {{- if $cd.Environment }}
+        environment: [
+            {{- range $key, $value := $cd.Environment }}
+            {
+                name: "{{ $value.Name }}",
+                value: {{ modelCase $value.Value }},
+            },
+            {{- end }}
+        ],
+        {{- end }}
+        {{- if $cd.DependsOn }}
+        dependsOn: [
+            {{- range $cd.DependsOn }}
+            {{- if $i }}
```

Comment:
Is this `$i` referring to the right thing? Or do you need to assign a range variable

Reply:
yeah i cleaned up this template, also tested ecs -> efs because there was a fix needed for that as well